polar-core/src/rewrites.rs

```diff
@@ -160,6 +160,14 @@ impl<'kb> Folder for Rewriter<'kb> {
         }
     }
 
+    fn fold_rest_variable(&mut self, v: Symbol) -> Symbol {
```

Comment:
Uh, I don't really get this, this looks like a function that isn't called anywhere?

Reply:
oh oh n/m I get it, lol was odd at first look didn't realize this was a method overload.